src/main/java/org/polystat/Analysis.java

```diff
@@ -45,4 +45,9 @@
     Iterable<String> errors(Func<String, XML> xmir,
         String locator) throws Exception;
 
+    /**
+     * Name of the analysis.
+     * @return Name of the analysis.
+     */
+    String name();
```

Comment:
@Sitiritis it seems that just the canonical name of the class (`org.polystat.far.Reverses`) is more than enough to describe the analyzer. I don't see what is the extra value in this `name()` method... Let's get rid of this.